### DISCUSSION TITLE
Welcome to sequel-pro-laravel-export Discussions!

### DISCUSSION THREAD

**Discussion by cviebrock** [cviebrock/sequel-pro-laravel-export#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by JamesNock** [comment]
This is an awesome tool, thank you for sharing it. I wonder if it could be extended or a similar one created to also create a seeder from the table's data?
